### DISCUSSION TITLE
blog/internet-observation-during-the-vaccination-period

### DISCUSSION THREAD

**Discussion by giscus** [Pathsis/2750-words#13] [Announcements]
# blog/internet-observation-during-the-vaccination-period

我们必须努力避免，在特定的时候，比如在防疫成效显著的时候，政府的威信得到了极大的提升，而在防疫遇到明显困难的时候，政府立即被一些人骂为匪帮。

https://rercel.com/blog/internet-observation-during-the-vaccination-period

**Comment by ghost** [comment]
去年我也想过关于SI YUE ZHI SHENG的问题，它的恶意性过于明显，为了摧毁现有的政治合法性和治理秩序；如果考虑公民对公共事务的参与，又如何看待台湾国内的民粹兴起和政治冷感（公投参与率低），反而依赖于“公知”，但从科学决策的角度来看，知识分子面向大众的线性传播是最为简单的，其实政策企业家（Policy entrepreneur）也可以参与进来，有待改善。

> 有一个朋友喜欢在朋友圈抒发心情。他暗示，疫情期间政府的作为让他有了厌世的情绪。他表示，自己早上醒来觉得在这样的世道活下去不如死了算了。


我在反送中期间有过这样的情绪，因为自己和他们共享一样的广府文化，每天一起来就是看youtube上多个电视台的现场直播，但到了冲击立法会的那日我觉得我错了，反对者也不尊重《逃犯条例》正在进行的二读，这件事情的性质发生改变；这种气氛很容易演变为仇恨、抑郁、愤怒，这在后来的暴力事件加剧体现出来，据说后来法官也给那些参与暴动的年轻人宽大处理。

到现在的《荣光》…说下民族认同缺失的角度，特别是本土文化的消逝，社会主流文化上的排斥，似乎不得不流放diaspora（香港人移民潮），但内心又很喜欢这种本土性，似乎更容易认同这种恶意呈现。
